### PR TITLE
Subtribe get

### PR DESCRIPTION
**Pull request by chereseeriepa**
Adds ssb.tribes.subtribes.get method

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -269,6 +272,18 @@ function init (ssb, config) {
           })
         })
       },
+      get: tribeGet,
+      // get (id, cb) {
```
Comment (chereseeriepa) [inline index.js:276]: confirm with @mixmix  first about whether we need to return the dmKey as well

Comment (mixmix) [reply index.js:275]: leave that to me - I need to put a PR in which deals with dmKeys

File: README.md [hunk 2]
```diff
@@ -160,6 +160,16 @@ where:
 
 _This method calls `ssb.tribes.create`_
 
+### `ssb.tribes.subtribe.get(groupId, cb)`
+
+Returns group metadata for a given group:
```
Comment (mixmix) [inline README.md:165]: :fire: just say
> _alias of `ssb.tribes.get`_

**Review by mixmix — CHANGES_REQUESTED** [review]
since it's an alias method, lets just say it's an alias.
I'm expecting to be able to support dmKeys for all groups as well, so expect this to remain an alias for the moment

**Pull request merged**